tasks/kibana.yml

```diff
@@ -1,15 +1,23 @@
 ---
-- name: Download Kibana tarball.
-  get_url:
-    url: https://download.elastic.co/kibana/kibana/kibana-{{ elk_kibana_version }}-linux-x64.tar.gz
-    dest: /opt/kibana-{{ elk_kibana_version }}-linux-x64.tar.gz
-    # Don't download file every time, only if it doesn't exist.
-    force: no
+- name: Add Kibana repo apt key.
+  apt_key:
+    url: https://packages.elastic.co/GPG-KEY-elasticsearch
+    state: present
 
-- name: Check for existing Kibana installation.
-  stat:
-    path: /opt/kibana-{{ elk_kibana_version }}-linux-x64
-  register: kibana_directory_check
+- name: Add Kibana apt repo.
+  apt_repository:
+    repo: "deb http://packages.elastic.co/kibana/4.5/debian stable main"
+    state: present
+  notify:
+    - reload systemd
+
+- name: Install Kibana.
```

Comment:
Out of scope for this role, since it's grsec-specific—we'll handle the change via vars, and the paxctld role will notify the handler when we run our logserver playbook.